CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Chrono-tz Changelog
 ===================
 
+## 0.9.0
+
+* **tzdb** Update tzdb from 2022f to 2023b. Some timezones have been removed. For
```

Comment:
The email doesn't mention anything about timezones being removed. Is this copy/pasta? If no timezones were removed, this can be 0.8.1 instead.

Reply:
oof; whoops. That was indeed copypasta. good catch.

Reply:
Yeah, adding links to both would be nice.